pkgs/by-name/br/brltty/package.nix

```
@@ -82,18 +84,29 @@ stdenv.mkDerivation rec {
     "install-polkit"
   ];
 
-  preConfigure = ''
-    substituteInPlace configure --replace /sbin/ldconfig ldconfig
+  preConfigure =
+    ''
+      substituteInPlace configure --replace "/sbin/ldconfig -n" "true"
 
-    # Some script needs a working tclsh shebang
-    patchShebangs .
+      # Some script needs a working tclsh shebang
+      patchShebangs .
 
-    # Skip impure operations
-    substituteInPlace Programs/Makefile.in    \
-      --replace install-writable-directory "" \
-      --replace install-apisoc-directory ""   \
-      --replace install-api-key ""
-  '';
+      # Skip impure operations
+      substituteInPlace Programs/Makefile.in    \
+        --replace-fail install-apisoc-directory ""   \
+        --replace-fail install-api-key ""
+    ''
+    + lib.optionalString (stdenv.system != buildPackages.stdenv.system) ''
+      # ./configure call itself second time for build platform, if it fail -- it fails silently, make it visible
+      # (this is not mandatory changing, but make further maintaining easier)
+      substituteInPlace mk4build \
+        --replace-fail "--quiet" ""
```

nit: I like having debug replaces like this only warn if they fail, that way the bot update won't get caught up on something that is only good for debugging, not affecting the actual build.
```suggestion
        --replace-warn "--quiet" ""
```